randovania/data/json_data/prime3.txt

```diff
@@ -3392,13 +4197,20 @@ Asset id: 13146584938063573409
 > Dock to Transit Station 1-A; Heals? False
   * Normal Door to Transit Station 1-A/Dock to Landing Site Alpha
   > Samus Ship
+      After Landing Site Alpha Barrier
+  > Event - Barrier
       Grapple Voltage
 
-> Samus Ship; Heals? False
+> Samus Ship; Heals? True
   > Dock to Scrapvault Lift
       Trivial
   > Dock to Transit Station 1-A
-      Grapple Voltage
+      After Landing Site Alpha Barrier
```

Comment:
Add another node that connects to the event with voltage as requirement.

Reply:
Another node? Would it make more sense to just have scrapvault lift and transit station 1-A both connect to Event - Barrier with voltage as the requirement?

Reply:
My bad I wrote that wrong. I meant have the ship connect to the event as well with voltage. a new node wouldnt make sense.